README.md

```diff
@@ -16,7 +35,11 @@ import ReactDOM from 'react-dom';
 import HeaderSmall from '@nib-components/react-header-small';
 
 ReactDOM.render(
-  <HeaderSmall/>,
+  <div>
+    <HeaderSmall/>
+    <HeaderSmall white />
```

Comment:
Did we consider naming this something like `theme="white"` or `color="white"` instead?